### DISCUSSION TITLE
Share multiple albums

### DISCUSSION THREAD

**Discussion by smolesen** [photoprism/photoprism#3170] [Q&A]
Is there an easy way to select multiple albums and share them with the same secret? or do I have to select the albums one by one, click share and enter the common secret.... quite cumbersome when having 200 albums

and also is there a way to turn multiple folders into multiple albums, without having to create them one-by-one?

TIA

/Søren

**Comment by graciousgrey** [comment]
> Is there an easy way to select multiple albums and share them with the same secret?

Not at the moment, but a lot of improvements are planned for sharing:

   https://github.com/photoprism/photoprism/issues/1162
   https://github.com/photoprism/photoprism/issues?q=is%3Aissue+is%3Aopen+sharing+in%3Atitle+

> and also is there a way to turn multiple folders into multiple albums, without having to create them one-by-one?

What exactly do you want to achieve with this? You can select folders and add all photos of the selected folders to one or multiple albums.

:point_right: https://docs.photoprism.app/user-guide/organize/folders/#create-albums-from-folders

**Reply by smolesen** [reply]
The thing is, that I pretty much want all my folders (200+) to be turned into albums, but it seems like I have to do it one by one. If I select multiple folders, they seem to be "merged" into one album....

I've looked at the DB schema, and it seems to be pretty easy, to create a program, that can create albums from folders one-to-one....
